tree/ntuple/v7/src/RField.cxx

```diff
@@ -1782,6 +1782,18 @@ void ROOT::Experimental::RField<std::string>::AcceptVisitor(Detail::RFieldVisito
 
 //------------------------------------------------------------------------------
 
+ROOT::Experimental::RClassField::RClassField(std::string_view fieldName, const RClassField &source)
+   : ROOT::Experimental::RFieldBase(fieldName, source.GetTypeName(), ENTupleStructure::kRecord, false /* isSimple */),
+     fClass(source.fClass),
+     fSubFieldsInfo(source.fSubFieldsInfo),
+     fMaxAlignment(source.fMaxAlignment)
+{
+   for (const auto &f : source.GetSubFields()) {
+      RFieldBase::Attach(f->Clone(f->GetFieldName()));
+   }
+   fTraits = source.GetTraits();
```

Comment:
Most other implementations do this in `CloneImpl()` and just forward the cloned subfield(s). I don't mind either way, but maybe we should try to be consistent...

Reply:
As discussed offline, we go with this for the moment to avoid constructing and moving vectors around. We have to make another pass to improve consistency afterwards.